bus/routes.go

```diff
@@ -1778,6 +1772,85 @@ func (b *Bus) slabsPartialHandlerPOST(jc jape.Context) {
 	})
 }
 
+func (b *Bus) autopilotHandlerGET(jc jape.Context) {
+	ap, err := b.store.Autopilot(jc.Request.Context())
+	if jc.Check("failed to fetch autopilot", err) != nil {
+		return
+	}
+
+	// sanity check autopilot config is valid
+	if ap.Enabled {
```

Comment:
I considered doing this when we update the config but in the end opted to log a warning and disable it on fetch. Not too sure what the best approach is here, I know `hostd` likes to not validate its config when it's not enabled anyway, that means you can't trust the database has valid config though. Which means you'd have to validate it when you enable it and return a 409 or something when you find the persisted config is invalid.

Reply:
`hostd` likes to what? Config is always validated before it hits the database in `hostd` and the database stores its config as individual columns instead of unstructured key value blobs. If the database is in a working state, the config must be valid.

The only reason this would be necessary would be if you're not validating it in the setter, the blob format changes without a migration, or you're expecting the user to go and mess with the database manually.

Reply:
I wasn't trying to offend `hostd` here. We're migrating the `autopilot` from unstructured blobs to individual columns, essentially copying `hostd`'s way of persisting config and settings. I remember you were annoyed once with a `renterd` endpoint where you were trying to update a setting and it was returning an error even though the feature was disabled. I'll update the endpoint.